build_ngtf.py

```diff
@@ -272,10 +282,6 @@ def main():
                           "https://github.com/tensorflow/tensorflow.git",
                           tf_version)
             os.chdir(pwd_now)
-            if not arguments.disable_cpp_api:
-                # Now build the libtensorflow_cc.so - the C++ library
-                build_tensorflow_cc(tf_version, tf_src_dir, artifacts_location,
-                                    target_arch, verbosity)
         else:
```

Comment:
Is this still needed to build CC API? How would someone build the CC API with prebuilt TF now?

Reply:
As discussed, I removed the dependence for prebuilt option on this flag to reduce the number of build flags we need to pass.